### PR TITLE
Updated url-utils to appent the path by extension without using private APIs that have since changed.

### PR DESCRIPTION
**Pull request by usergenic**
This fix was part of https://github.com/Polymer/tools/pull/442 but that PR is stalled due to a problem with wct-local not shutting down selenium servers and the whole thing, well... stalling when integration tests run.  Need to fix that before PR can merge.

### CODE REVIEW

File: packages/bundler/src/url-utils.ts [hunk 1]
```diff
@@ -27,10 +27,11 @@ import constants from './constants';
  *     'file:///something/something.html_omg.js?ponies'
  */
 export function appendUrlPath(url_: string, extension: string): string {
-  const uri = Uri.parse(url_);
-  uri['_path'] = `${uri.path}${extension}`;
-  return uri.toString();
+  const {scheme, authority, path, query, fragment} = Uri.parse(url_);
```
Comment (justinfagnani) [inline packages/bundler/src/url-utils.ts]: Is there a reason to list all the individual fields? How do we know we got all the important ones? I'd feel more confident with just updating the path...

Comment (usergenic) [reply packages/bundler/src/url-utils.ts]: https://github.com/Microsoft/vscode-uri/blob/master/src/index.ts#L684

The reason to list them is that technically these are the only fields that should be in the input to `Uri.from()` whereas the Uri instance returned from a parse call contains more stuff according to typings and then we get compile errors.

Comment (usergenic) [reply packages/bundler/src/url-utils.ts]: Can get around it with Object.assign but that's only because its a typings cheat.

Comment (usergenic) [reply packages/bundler/src/url-utils.ts]: And object spread syntax returns error about unsupported fields like fsPath

Comment (justinfagnani) [reply packages/bundler/src/url-utils.ts]: ok, makes sense if the types are different.

**Comment by usergenic** [comment]
Due to travis being delayed, the other PR actually finally got past its CI and merged before this one did.  So just closing this out.

**Pull request closed without merging**